### PR TITLE
Milestone 9. Popup window details

### PR DESCRIPTION
**Pull request by indigodavid**
The current milestone adds a popup window and the following features:

- Js objects that generate the Projects and Featured Project sections from an array
- Modal window to display the highlighted project
- New links for each project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -541,3 +543,75 @@ nav {
...
+.modal-container {
+  display: none;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.6);
+  inset: 0 0;
+  width: 100vw;
+  height: 100vh;
+  padding: 10% 0;
+  margin: 0;
+}
```
Comment (thecodechaser) [inline css/style.css]: - [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please reduce the top margin and padding from all sides for the model and make sure it matches the Figma design. Please add a blur effect in the background.
  - For blur you can use effect `backdrop-filter: blur(5px);`, use px according to your design.

### Your design

image

### Figma design

image

File: css/style.css [hunk 2]
```diff
@@ -541,3 +543,75 @@ nav {
...
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-close {
+  font-size: 28px;
+  color: #67798e;
+}
```
Comment (thecodechaser) [inline css/style.css]: - [ ] Please increase the close(X) button weight and make sure it matches the design.

File: css/style.css [hunk 3]
```diff
@@ -541,3 +543,75 @@ nav {
...
+  background-color: #fff;
+  width: 90vw;
+  height: auto;
+  padding: 5%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 16px;
+}
```
Comment (thecodechaser) [inline css/style.css:571]: - [ ] Please make sure the project description is aligned at the project's image start point and model buttons are aligned at the project's image end point and please make sure it matches with the design.

### Your design

image

### Figma design

image

File: css/style.css [hunk 4]
```diff
@@ -541,3 +543,75 @@ nav {
...
+  cursor: pointer;
+}
+
+#live-link {
+  background-image: url(../images/see-live.svg);
+}
+
+#source-link {
+  background-image: url(../images/github.svg);
+}
```
Comment (thecodechaser) [inline css/style.css:603]: - [ ] Please increase the font-weight for model buttons and reduce the gap between buttons and make sure it matches with Figma design.

### Your design

image

### Figma design 

image

File: css/style.css (hunk 1 above)
Comment (thecodechaser) [inline css/style.css]: - [ ] If the screen is smaller than the model, The complete model is not visible on the screen and your model is not scrollable so users can not see the complete model. Please use `overflow: scroll;` in your model controller so users can scroll the model.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @indigodavid👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @indigodavid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**